manager/integration/tests/test_upgrade.py

```diff
@@ -318,8 +421,9 @@ def test_upgrade(longhorn_upgrade_type,
             new_ei = ei
 
     for v in volumes:
-        volume = client.by_id_volume(v.name)
-        volume.engineUpgrade(image=new_ei.image)
+        if v.name != restore_vol_name:
```

Comment:
Forgot the question. Why do we need this non-equal check?

Reply:
The volume is created after the upgrade. The engine has already been the latest. Call `engineUpgrade` again on it will throw exception.

Reply:
I see. Thanks for the clarification.